### DISCUSSION TITLE
Welcome to xplorer Discussions!

### DISCUSSION THREAD

**Discussion by kimlimjustin** [kimlimjustin/xplorer#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avi12** [comment]
This seems like a cool project! Could it integrate directly on Windows, replacing the native File Explorer?

**Reply by kimlimjustin** [reply]
Yes, you can follow the guide [here](https://xplorer.vercel.app/docs/guides/xplorer%20as%20default/). However, it is still pre release, make sure to read all the warnings on the docs.

**Reply by avi12** [reply]
I will consider integrating it when it's in a more stable release, one that can deliver at minimum the same functionality that I'm used to

**Reply by kimlimjustin** [reply]
> I will consider integrating it when it's in a more stable release, one that can deliver at minimum the same functionality that I'm used to

Sure, I recommend that too, just make it as the default when I release the stable version, you can `watch` this repo to get any update. Meanwhile, you can use Xplorer alongside with default file explorer too, I sure for basic usage, it will be okay, if something went wrong, please open an issue here.

However, if you know how to switching the default file explorer, you can directly play with it, you can change it back from xplorer to default explorer by changing the `openinxplorer` as mentioned on the step 5 to `open`.

**Reply by avi12** [reply]
Sounds good! Thanks for making this application!

**Reply by kimlimjustin** [reply]
Thanks, btw if you'd like to, any contributions will be welcomed :)
